src/dc_job.rs

```diff
@@ -821,7 +832,7 @@ pub unsafe fn dc_perform_imap_fetch(context: &Context) {
         context,
         0,
         "INBOX-fetch done in {:.4} ms.",
-        clock().wrapping_sub(start) as libc::c_double * 1000.0f64 / 1000000 as libc::c_double,
+        start.elapsed().as_millis(),
```

Comment:
as_millis returns *whole* number of milliseconds, but you format them as floating point anyway. Format them as an integer, convert them to nanoseconds and divide manually or enable experimental `duration_float` feature.

Reply:
makes sense to cleanup but i think it shouldn't hold up merging this PR as it's so big to easily cause conflicts ...

Reply:
fixed